### ISSUE TITLE
No Server Config File

### ISSUE THREAD

**Issue by Omegasutoraiki** [henkelmax/simple-voice-chat#191]
When I try to run the mod in a server it doesn't create a config file. This is an issue for me because we are unable to use the voice chat feature without it. We are all just constantly disconnected from the voice chat.
Is there a way to get the config? Or do I just have to take the client config, because I would really like to use the mod since my laptop hates using discord and Minecraft simultaneously in every version past 1.12.2

**Comment by henkelmax** [comment]
Have you read the FAQ?

**Comment by Omegasutoraiki** [comment]
Yes, I have looked all through the folders and cannot find the subfolder in world. I honestly have no clue what I should do.
I cannot find the ServerConfig subfolder that the FAQ says should be there.

**Comment by henkelmax** [comment]
Which modloader are you using?

**Comment by Omegasutoraiki** [comment]
Forge

**Comment by henkelmax** [comment]
Can you send me a screenshot of your world folder?

**Comment by Omegasutoraiki** [comment]
![wrld](https://user-images.githubusercontent.com/47616823/119724744-b5a50600-be34-11eb-9d66-856024bd61a1.PNG)

**Comment by henkelmax** [comment]
Can you send me the logs?

**Comment by Omegasutoraiki** [comment]
Yea, sorry if I take awhile.

**Comment by henkelmax** [comment]
No worries

**Comment by Omegasutoraiki** [comment]
[2021-05-24-1.log](https://github.com/henkelmax/simple-voice-chat/files/6549561/2021-05-24-1.log)
[2021-05-24-3.log](https://github.com/henkelmax/simple-voice-chat/files/6549562/2021-05-24-3.log)
[2021-05-24-2.log](https://github.com/henkelmax/simple-voice-chat/files/6549563/2021-05-24-2.log)

**Comment by henkelmax** [comment]
I can't see any traces of the voice chat mod in your logs. It also seems your logs are incomplete

**Comment by Omegasutoraiki** [comment]
Weird. I have been gone the last few days. Maybe the server just didn't pick up the mod in the folder or something.

**Comment by henkelmax** [comment]
Please make sure everything is installed correctly.

**Comment by Omegasutoraiki** [comment]
I fixed the issue I was having. I looked into it more than I had before because of all this. I found out that the person who set it up didn't set the server to run through the Forge launcher and instead the normal Server Launcher. I still had to make a separate server to get the files required but after that it ran as it should. Thanks for replying to me, I probably would have given up on this otherwise.
Sorry for having to ask in the first place though. I hope you have a good day.
